src/config.ts

```diff
@@ -1,5 +1,6 @@
 import fs from 'fs';
 import yaml from 'js-yaml';
+import _ from "lodash";
```

Comment:
naming this import `_` is hard to understand later in the code. How about `import { merge } from "loadsh";`?

Reply:
Oh I see using `_` seems to be common for lodash https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark. So then its fine

Reply:
yup it's the standard